frontend/src/routes/Applications/Overview.tsx

```diff
@@ -917,7 +917,7 @@ export default function ApplicationsOverview() {
                 acmExtensions.applicationAction.forEach((appAction) => {
                     if (appAction?.model ? isResourceTypeOf(resource, appAction?.model) : isOCPAppResource(resource)) {
                         const ModalComp = appAction.component
-                        const close = () => setPluginModal(<ModalComp isOpen={false} />)
+                        const close = () => setPluginModal(<></>)
```

Comment:
This change will always open a fresh modal, without this model loading with previous state values.